snomed/com.b2international.snowowl.validation.snomed/src/test/java/com/b2international/snowowl/validation/snomed/GenericValidationRuleTest.java

```diff
@@ -214,17 +216,25 @@ public void rule665() throws Exception {
 	public void rule666() throws Exception {
 		final String ruleId = "666";
 		indexRule(ruleId);
+		
+		Function<SnomedDescriptionIndexEntry, SnomedRefSetMemberIndexEntry> languageRefsetMember = description -> {
```

Comment:
A helper method with the name of createLanguageRefSetMember would be much readable than using Functions/Lambdas.